unit_test/fuzzing/test_responder/test_spdm_responder_algorithms/algorithms.c

```diff
@@ -310,7 +310,7 @@ void libspdm_test_responder_algorithms_case10(void **State)
                                     spdm_test_context->test_buffer, &response_size, response);
 }
 
-void libspdm_test_responder_algorithms_case11(void **State)
+void libspdm_test_responder_algorithms_case12(void **State)
 {
```

Comment:
what is the difference between `libspdm_test_responder_algorithms_case11` and `libspdm_test_responder_algorithms_case12`?
why you rename here?
can you add some comment for the new test case to describe its test scenario?